### PR TITLE
Bookstore: add reducers and actions #2

### PR DESCRIPTION
**Pull request by ksigei**
### In this milestone, I was able to:

1. Add React Redux to my project
2. Structure my application files using a "feature folder" approach and use the ducks pattern for my Redux files.
3. Write my book's actions and reducer using ducks pattern

- Define action types for adding and removing a book.
- Set the initial state to be an empty array of books.
- Export Action Creators for my actions.
- Write your reducer and export it as default.

- [ ] Define state changes for the actions that you created.
- [ ] In case of unknown action - return the current state.
4. Write my categories actions and reducer using ducks pattern
- Define an action type for checking the status.
- Set the initial state to be an empty array of categories.
- Export Action Creators for your actions.
- Write my reducer and export it as default.

- [ ] For the check status action return a string "Under construction".
- [ ] In case of unknown action - return the current state.
5. Configure the Redux Store

- Import the necessary methods from Redux.
- Combine both reducers into a root reducer by using combineReducers function.
- Use the root reducer to create your store.

### CODE REVIEW

File: src/redux/Books/books.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
```
Comment (Meltrust) [inline src/redux/Books/books.js]: - [ ] These action types need to follow the duck's pattern, according to the project requirements:

image


Hint:  Examples of duck's pattern would be: 

```javascript
const ADD_BOOK = bookstore/books/ADD_BOOK 
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```

File: src/redux/Categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const CHECK = 'CHECK';
```
Comment (Meltrust) [inline src/redux/Categories/categories.js]: - [ ] These action types need to follow the duck's pattern, according to the project requirements:

image

Hint:  Examples of duck's pattern would be: 

```javascript
const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @ksigei,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice reducers and Redux config✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @ksigei, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**